tests/unit/date/test_util.py

```diff
@@ -86,3 +86,23 @@ def test_to_dt_dt_tz():
 
 def test_to_dt_dt_tz_default():
     assert to_dt(dt(1970, 1, 1, tzinfo=mktz('UTC')), mktz('Europe/London')) == dt(1970, 1, 1, tzinfo=mktz('UTC'))
+
+
+def test_daterange_raises():
+    with pytest.raises(ValueError):
+        assert(DateRange(dt(2013, 1, 1), dt(2000, 1, 1)))
+
+
+def test_daterange_eq():
+    dr = DateRange(dt(2013, 1, 1))
+    assert((dr == None) == False)
```

Comment:
`assert` is a statement (in both python 2 & 3),  and since the statement inside evaluates to a single `True` or `False`, the outer bracket pair is redundent.

(applies to all the `assert` in this PR)